templates/Package.swift

```diff
@@ -25,7 +25,7 @@ let package = Package(
                 .target(name: "RustFramework")
             ]{% if disable_warnings %},
             swiftSettings: [
-                .unsafeFlags(["-suppress-warnings"]),
+                .define("SWIFT_PACKAGE_MANAGER"),
```

Comment:
Could you link a source that explains what this define does? A quick search did not yield meaningful results. Thanks!

Reply:
To be honest, i found this somewhere as a quick fix. The only meaningful source i could found for it is the source code itself (see below). However, on my current setup i removee the whole `swiftSettings` flag and the code builds successfully without warnings, so maybe its unnecessary. 

[Here i found the first report ](https://forums.swift.org/t/override-for-unsafeflags-in-swift-package-manager/45273/4) of the issue, with a workaround which sounds like a hack. If you know another way of getting the build succeed please let me know! For now i remove the `swiftSettings` flag, reset the Packages Cache in Xcode and then it builds. 

`/// The following example shows how to use build setting conditions with various
/// APIs:
///
/// ```swift
/// ...
/// .target(
///     name: "MyTool",
///     dependencies: ["Utility"],
///     cSettings: [
///         .headerSearchPath("path/relative/to/my/target"),
///         .define("DISABLE_SOMETHING", .when(platforms: [.iOS], configuration: .release)),
///     ],
///     **swiftSettings: [
///         .define("ENABLE_SOMETHING", .when(configuration: .release)),
///     ],**
///     linkerSettings: [
///         .linkLibrary("openssl", .when(platforms: [.linux])),
///     ]
/// ),
/// ````

Reply:
Ok, I think I will apply the solution currently present on main then (not using the suppressWarnings flag by default and include an option that suppresses them)